FHIRClientDevelopmentGuide.md

```diff
@@ -0,0 +1,100 @@
+# FHIR Client Development Guide
+November 17, 2022
+
+[Introduction](#introduction)  
+[Integration with EDRS System](#integration-with-edrs-system)  
+[Authentication](#authentication)  
+[Timeliness](#timeliness)  
+[Message Resends](#message-resends)  
+[Response Message Handling](#response-message-handling)  
+[Message Traceabality](#message-traceabality)  
+[NCHS Backup Endpoint](#nchs-backup-endpoint)  
+
+## Introduction
+The NVSS FHIR Modernization effort will implement the exchange of FHIR messages between jurisdiction’s and NCHS for mortality data. NCHS has implemented a test FHIR API server for jurisdictions to submit their mortality data. Each jurisdiction will implement a client that exchanges FHIR data with the NCHS server. A jurisdiction’s client implementation must follow the messaging protocol described in the [FHIR Messaging IG](http://build.fhir.org/ig/nightingaleproject/vital_records_fhir_messaging_ig/branches/main/message.html) to ensure reliable message delivery and traceability. The [Reference Client](https://github.com/nightingaleproject/Reference-Client-API) is an open-source example of a C# implementation of the FHIR Messaging IG. Implementers may leverage the Reference Client, tools such as Rhapsody, or develop a custom solution to communicate with the API. In general, each Client implementation should support the following guidelines when deployed to production. Each section species configurable parameters that should be defined in a separate config file.
```

Comment:
Should the word "test" be removed from "test API" since the document talks about moving to production? And maybe this section should touch on the STEVE API / NCHS API explanation?